app.js

```diff
@@ -50,7 +68,21 @@ console.log(
 //    that checks if the year is a leap year, and returns the correct
 //    number of days for Feb. The default value for the year should be
 //    the current year
-function daysInMonth() {}
+function daysInMonth(monthNum) {
+    if (monthNum >= 1, monthNum <= 12){
+        if(monthNum === 2){
+            return 29
+        }
+        if(monthNum === 3){
+            return 31
+        }
+        if(monthNum === 11){
```

Comment:
These definitely work for what is getting logged below, but I'd like to see you handle the other months as well.